src/extension.ts

```diff
@@ -429,6 +430,7 @@ export namespace Openshift {
                     if (result.stderr === "") {
                         explorer.refresh();
                         const logoutInfo = await vscode.window.showInformationMessage(`Successfully logged out. Do you want to login to a new cluster`, 'Yes', 'No');
+                        vscode.commands.executeCommand('setContext', 'isLoggedIn', false);
```

Comment:
Just a minor nitpick - please put this line before the info message so the log out option disappears along with the cluster.

Reply:
Fixed this.